Sources/Application/AppDelegate.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  AppDelegate.swift
+//  di-mobile-ios-onelogin-app
+//
+//  Created by Amoakohene, Magdaline on 24/08/2023.
+//
```

Comment:
Are we able to remove these autogenerated headers from the `.swift` files please?